### ISSUE TITLE
Allow updating min/max values of ofxSliderGroup

### ISSUE THREAD

**Issue by woutgg** [openframeworks/openFrameworks#6378]
Currently there is no way to update the min/max values for an ofxSliderGroup like with the 'regular' ofxSlider. I have a vec2 slider which range is dependent on another slider so I need to be able to update it dynamically.

I don't think adding this has any drawbacks so if there are no objections I could submit a PR.

Of course, it would even be nicer if the sliders could automatically update whenever min/max values change on the underlying parameter since (to me at least) that is the actual data, the sliders only being a view upon them.

**Comment by roymacdonald** [comment]
Hi, you actually can. If you use an `ofParameter<glm::vec2>` to create the gui element like
```
// in ofApp.h
ofxPanel gui;
ofParameter<glm::vec2> param = {"Some parameter", {0,0}, {0,0}, {10,10} }; 

// in ofApp.setup(). Just the regular setup using ofParameter instead of ofxSliderGroup
gui.add(param);

// then if you want to change the min or max, call
param.setMin(glm::vec2(x_value, y_value));// for x_value and y_value you can use another sliders values

```

**Comment by roymacdonald** [comment]
This is not true. Just double checked and it is not working with :( 
I am taking a look on how to fix this.

**Comment by roymacdonald** [comment]
This is a bug, as there is a function for setting the min and max it will not reflect on the gui. 
I am taking a look at it

**Comment by roymacdonald** [comment]
right. and this fix you made is working? 
if so, can you please make a PR?

**Comment by woutgg** [comment]
Yup it works, although it only implements half of the 'ofParameter <=> slider' coupling behaviour, so I think it'd be good to document that discrepancy compared to `ofxSlider`.

I'll happily submit a PR though :), somewhere this weekend probably.

**Comment by roymacdonald** [comment]
Hi, I think that instead of documenting this discrepancy it should get fixed so it behaves in the same way as ofxSlider. I think that this issue also shows up on other sliders like the color or rectangle sliders. I'll take a look at it on the next days. 
best

**Comment by woutgg** [comment]
Sorry for the delay in submitting the PR, let me know if I can help out finishing this.
